test/rbi/rbs_printer_test.rb

```diff
@@ -398,6 +401,10 @@ def foo: -> void
 
         # @override(allow_incompatible: true)
         def bar: -> String?
+
+        # @without_runtime
```

Comment:
Why do we want to remember if a signature had originally been marked as without runtime? To translate back?